homeassistant/components/light/qwikswitch.py

```diff
@@ -0,0 +1,37 @@
+"""
+Support for Qwikswitch Relays and Dimmers as HA Lights.
+
+See the main component for more info
+"""
+import logging
+from homeassistant.components.qwikswitch import QSToggleEntity, QSUSB

```

Comment:
So to fix, is it ok if I just `import homeassistant.components.qwikswitch` and then refer to `qwikswitch.QSUSB` and `qwikswitch.QSToggleEntity`?

I seem to remember pylint didnt like globals being defined in the main scope, only `None`. That is why I define it as an empty list if its `None`